src/modules/client/components/planning/Planning.vue

```diff
@@ -10,7 +10,7 @@
       <div class="col-xs-12 col-md-7">
         <planning-navigation :timeline-title="timelineTitle()" :target-date="targetDate" :type="PLANNING"
           @go-to-next-week="goToNextWeek" @go-to-previous-week="goToPreviousWeek" @go-to-today="goToToday"
-          @go-to-week="goToWeek" :is-coach-or-planning-referent="isCoach || isPlanningReferent"
+          @go-to-week="goToWeek" :is-coach-or-planning-referent-or-auxiliary="isCoach || isPlanningReferentOrAuxiliary"
```

Comment:
je pense qu'on peut raccorcir en is-coach-or-auxiliary

Reply:
donne